app/api/serializer_fields.py

```diff
@@ -0,0 +1,25 @@
+from django.core.exceptions import ValidationError
+
+from rest_framework.fields import Field
+
+import jsonschema
+from jsonschema.exceptions import SchemaError
+
+class JsonBField(Field):
+    """ Custom serializer class for JsonB
+
+    Do no transformations, as we always want python dicts
+    Validate schema when converted to internal representation
+
+    """
+    type_name = 'JsonBField'
+
+    def to_representation(self, value):
+        return value
+
+    def to_internal_value(self, value):
```

Comment:
We could make a `JsonSchemaField` that inherits from this though and this would work for that.